lib/fathead/npm_cli/cover/redirects.txt

```diff
@@ -0,0 +1,74 @@
+npm access public, access
+npm access restricted, access
```

Comment:
This looks good! I'm not sure if `npm` needs to be there though because it is a trigger word. @moollaza?